model/task/db.go

```diff
@@ -666,6 +666,34 @@ func FindOneIdWithFields(id string, projected ...string) (*Task, error) {
 	return task, nil
 }
 
+func findAllTaskIDs(q db.Q) ([]string, error) {
+	tasks := []Task{}
+	err := db.FindAllQ(Collection, q, &tasks)
+	if err == mgo.ErrNotFound {
+		return nil, nil
+	}
+	if err != nil {
+		return nil, errors.Wrap(err, "error finding task ids for versions")
+	}
+
+	ids := []string{}
+	for _, t := range tasks {
+		ids = append(ids, t.Id)
+	}
+
+	return ids, nil
+}
+
+func FindAllTaskIDsFromVersion(versionId string) ([]string, error) {
```

Comment:
nits, but can you change uses of "Id" to "ID"

Reply:
I kept it like this to match this package. `BuildIdKey`, for example, is Id.